.devcontainer/bazel-base/Dockerfile

```diff
@@ -48,7 +48,7 @@ RUN echo "Install general purpose packages" && \
         libsctp-dev \
         libssl-dev \
         # dependency of pip systemd
-        libsystemd-dev \
+        libsystemd-dev=245.4-4ubuntu3.18 \
```

Comment:
The PR title says "Backport fixes to v1.8", but this doesn't seem to be on master.

Reply:
Why do we need exactly need this version?

Reply:
This is due to an issue with the package present on the artifactory.  `libsystemd-dev` currently uses the `245.4-4ubuntu3.17` from the artifactory. It needs the matching version for `libsystemd0`, which is not on the artifactory and has been removed from the ubuntu archive. 


https://launchpad.net/ubuntu/+source/systemd
https://artifactory.magmacore.org/ui/packages/deb:%2F%2Flibsystemd-dev?name=libsystemd&type=packages